### DISCUSSION TITLE
docker container with OpenZiti support

### DISCUSSION THREAD

**Discussion by dovholuknf** [lf-edge/ekuiper#2758] [General]
Hi all, https://github.com/lf-edge/ekuiper/pull/2603 added support for OpenZIti and at the bottom @ngjaying agreed to deploy a version of ekuiper with the changes included.

Could you please let me know if that was released and if so what the version number is I can refer to? 

TIA

**Comment by dovholuknf** [comment]
Welp -- it looks like it was tagged and I didn't notice!!! :) from that thread:
@dovholuknf https://github.com/lf-edge/ekuiper/releases/tag/v1.14.0-alpha.2 tagged.

**Comment by dovholuknf** [comment]
@ngjaying -- is there a way to override the property file from outside the docker container? In [this commit](https://github.com/lf-edge/ekuiper/pull/2603/commits/85c3e45f6c1153c91688e7467053f8ccefa20c50) you added this gate:
```
if conf.Config != nil && conf.Config.Basic.EnableOpenZiti == true {
```
I'm trying to figure out how I can set that to true from outside the docker container without mounting in some new property file. I'll be trying to look through to doc to see if I can do this via environment variable or some other mechanism.

TIA

**Reply by ngjaying** [reply]
Check here: https://ekuiper.org/docs/en/latest/configuration/configuration.html#environment-variable-syntax.  Should be KUIPER__BASIC__ENABLEOPENZITI

**Comment by dovholuknf** [comment]
*much appreciated*! that was what I was missing. thank you!
